portal/server/routes/utilisation-report-service/utilisation-report-download/index.js

```diff
@@ -1,15 +1,16 @@
 const express = require('express');
-const { validateBankIdForUser, validateMongoId, validateToken, validateRole } = require('../../middleware');
-const { ROLES: { PAYMENT_REPORT_OFFICER } } = require('../../../constants');
+const { validateBankIdForUser, validateToken, validateRole } = require('../../middleware');
+const {
+  ROLES: { PAYMENT_REPORT_OFFICER },
+} = require('../../../constants');
 const { getReportDownload } = require('../../../controllers/utilisation-report-service');
 
 const router = express.Router();
 
 router.get(
-  '/banks/:bankId/utilisation-report-download/:_id',
+  '/banks/:bankId/utilisation-report-download/:id',
```

Comment:
Can we validate the id here as it's a user input? Just the sql id validation would be enough, just as we've lost the validateMongoId now